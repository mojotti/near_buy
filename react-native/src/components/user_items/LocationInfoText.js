import React from 'react';
import { Text, View } from 'react-native';
import { styles } from '../../static/styles/UserItemDetailsStyles';

export default class LocationInfoText extends React.Component {
  render() {
    return (
      <View style={{ alignSelf: 'stretch' }}>
        <Text
          style={styles.infoText}
        >{`\u2022 Your exact location is not visible to others`}</Text>
        <Text style={styles.infoText}>
          {`\u2022`}
          <Text style={{ color: 'blue' }}> Blue</Text>
          <Text>{` color indicates your item's location`}</Text>
        </Text>
        <Text style={styles.infoText}>
          {`\u2022`}
          <Text style={{ color: 'red' }}> Red</Text>
          <Text>{` color indicates your location`}</Text>
        </Text>
      </View>
    );
  }
}
