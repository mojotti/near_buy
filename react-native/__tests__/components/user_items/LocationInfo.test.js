import React from 'react';
import { shallow } from 'enzyme';
import LocationInfoText from '../../../src/components/user_items/LocationInfoText';

describe('<LocationInfoText />', () => {
  test('renders correctly', () => {
    expect(shallow(<LocationInfoText />)).toMatchSnapshot();
  });
});
