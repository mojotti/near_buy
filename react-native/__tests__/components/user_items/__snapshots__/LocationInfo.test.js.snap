// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationInfoText /> renders correctly 1`] = `
<Component
  style={
    Object {
      "alignSelf": "stretch",
    }
  }
>
  <Component
    style={
      Object {
        "fontFamily": null,
        "marginBottom": 5,
        "marginLeft": 90,
        "textAlign": "left",
      }
    }
  >
    • Your exact location is not visible to others
  </Component>
  <Component
    style={
      Object {
        "fontFamily": null,
        "marginBottom": 5,
        "marginLeft": 90,
        "textAlign": "left",
      }
    }
  >
    •
    <Component
      style={
        Object {
          "color": "blue",
        }
      }
    >
       Blue
    </Component>
    <Component>
       color indicates your item's location
    </Component>
  </Component>
  <Component
    style={
      Object {
        "fontFamily": null,
        "marginBottom": 5,
        "marginLeft": 90,
        "textAlign": "left",
      }
    }
  >
    •
    <Component
      style={
        Object {
          "color": "red",
        }
      }
    >
       Red
    </Component>
    <Component>
       color indicates your location
    </Component>
  </Component>
</Component>
`;
